hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapLoaderTest.java

```diff
@@ -172,16 +171,13 @@ public void run() {
         }, 10);
     }
 
-    // ignored due to: https://github.com/hazelcast/hazelcast/issues/5035
-    @Ignore
     @Test
-    public void testMapLoaderLoadUpdatingIndex() throws Exception {
+    public void testMapLoaderLoadUpdatingIndex_noPreload() throws Exception {
```

Comment:
I thought this was the whole point of the test: Have a map with a MapLoader with _some_ settings, then destroy the map and re-create it with _different_ settings. At least this has always been my understanding of this test. 

It's failing due the racy nature of the IMap lifecycle, but it does not make it an invalid test?